static/js/groupDetails.js

```diff
@@ -0,0 +1,88 @@
+function remove_user_from_group(membership_id) {
+  showRemoveMemberModal(membership_id)
+}
+
+function remove_user_from_user_confirmed(btnElem) {
+
+  const membership_id = $(btnElem).attr("membership_id");
+  url = "/group/removeGroupMember";
+  var csrf_token = $('input[name="csrfmiddlewaretoken"]').val();
+  $.ajax({
+    url: url,
+    type: "POST",
+    data: { "membershipId": membership_id, 'csrfmiddlewaretoken': csrf_token},
+    success: function(data) {
+      $(`td[remove_user_button_id='${membership_id}-remove-user-button']`).html("");
+      $(`td[user_state_id='${membership_id}-user-state']`).html("Revoked");
+      checkbox_elem = $(`input[user_checkbox_id='${membership_id}-user-checkbox']`);
+
+      $(checkbox_elem).prop("checked", false);
+      $(checkbox_elem).prop("disabled", true);
+      showNotification("success", "Request sent successfully!")
+    },
+    error: function(jqXHR, exception) {
+      showNotification("failed", "Failed to send the request", "Something went wrong");
+    },
+  });
+
+  closeRemoveMemberModal();
+}
+
+function showRemoveMemberModal(membership_id) {
+  $('#remove_member_modal').show();
+  $('button[name="final-remove-button"]').attr("membership_id", membership_id)
```

Comment:
this one can be directly be found by ID right

Reply:
And instead of setting membership_id attribute you can also set onclick function to have membership id as input but you decide what to do